test/e2e/operator_test.go

```diff
@@ -905,3 +937,7 @@ func createTestWorkload(t *testing.T, namespace string) (*corev1.Service, *corev
 
 	return echosvc, ns
 }
+
+func isOnROSA() bool {
```

Comment:
```suggestion
func isOnROSA() bool {
    t.Helper()
```

Reply:
Same answer as in [this comment](https://github.com/openshift/aws-load-balancer-operator/pull/92#discussion_r1138447991).